html-web/html-and-the-web_div-salad/index.html

```diff
@@ -13,59 +13,59 @@
   </head>
   <!--someone forgot to wrap the content of our webpage with an HTML element,
 	which one was it again?-->
-  <div>
-    <div>
-      <div>Jane Doe</div>
+  <body>
+    <section>
+      <header>Jane Doe</header>
       <!--Which elements could be used to link to parts of the webpage,
 		which appear below?-->
-      <div id="nav">
-        <div>About Me</div>
-        <div>Contact</div>
-        <div>Skillset</div>
-      </div>
-    </div>
+      <nav id="nav">
+        <a href="#about-me">About Me</a>
+        <a href="#contact">Contact</a>
+        <a href="#skills">Skillset</a>
+      </nav>
+    </section>
     <!--Looks like the following part is the central content of our webpage.-->
-    <div>
-      <div id="about-me">
-        <div>About me</div>
-        <div>
+    <main>
+      <nav id="about-me">
+        <h1>About me</h1>
+        <p>
           Hi there! Welcome to my personal website. Here is gathered the most
           important information about me. If you want to work with me or get to
           know me, please don't hesitate to get in contact.
-        </div>
+        </p>
         <!--What element could be used to combine an image with a descriptive caption?-->
-        <div>
+        <figure>
           <img
             src="https://source.unsplash.com/random/400x300?fruit"
             alt="a random picture"
           />
-          <div>
+          <figcaption>
             A random picture, because I would like to maintain my privacy and
             let my code speak for me, instead of images.
-          </div>
-        </div>
-      </div>
+          </figcaption>
+        </figure>
+      </nav>
 
-      <div id="contact">
-        <div>Contact</div>
+      <nav id="contact">
```

Comment:
The same, the content inside a `nav` element should be of navigation. When not, we use `section`, `article`..